app/agents/voice/automatic/services/mcp/breeze_mcp_client.py

```
@@ -68,21 +65,13 @@ async def init_breeze_mcp_tools(
-                llm.register_function(name, function)
-
         # Log registration success
         if tools:
             logger.info(
                 f"Successfully registered MCP tools via pure Pipecat MCP client"
             )
         else:
-            logger.warning(
-                f"MCP client returned None or empty tools object - but essential tools are still registered"
-            )
+            logger.warning(f"MCP client returned None or empty tools object")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)